Library/TeamTalk.NET/TeamTalk.cs

```diff
@@ -3496,7 +3496,27 @@ public enum ClientEvent : uint
          * Tells how much audio remains in queue. The queue should 
          * be refilled as long as the audio input should remain active.
          */
-        CLIENTEVENT_AUDIOINPUT = CLIENTEVENT_NONE + 1080,            
+        CLIENTEVENT_AUDIOINPUT = CLIENTEVENT_NONE + 1080,
+        /**
+        * @brief The first voice packet of a new voice stream
+        * has been received.
+        *
+        * This time of this event may differ significantly from the start
+        * of the voice playout that is notified via
+        * #CLIENTEVENT_USER_STATECHANGE due to jitter buffering.
+        *
+        * The time between #CLIENTEVENT_USER_FIRSTVOICESTREAMPACKET
+        * and #CLIENTEVENT_USER_STATECHANGE is the fixed jitter delay
+        * configuration plus the currently active adaptive jitter
+        * buffering in the nActiveAdaptiveDelayMSec member of the
+        * User struct
+        *
+        * @param nSource 0
```

Comment:
Can 'nSource' be the stream-ID? Because then we can map the new voice stream in CLIENTEVENT_USER_AUDIOBLOCK.

Reply:
I can do it if you just say ok.

Reply:
Yes, you're right. Will you add it?

Reply:
yup

Reply:
I'll modify .NET and Java API tomorrow